docs/spelling_wordlist

```diff
@@ -455,6 +455,8 @@ Magee
 makemessages
 makemigrations
 Mako
+manouche
+Manouche
```

Comment:
I think the uppercased version isn't required in addition to the lower cased version.